### PR TITLE
Correctly support indefinite durations

### PR DESCRIPTION
**Pull request by manuelroemer**

### CODE REVIEW

**Comment by manuelroemer** [comment]
Should we also increment the package version/create a new release? After all, some flows of the last release have been fixed (e.g. revising orders, having an indefinite duration, ...).

File: src/order-basket/drug-search.ts [hunk 1]
```diff
@@ -69,7 +69,7 @@ function* explodeDrugResultWithCommonMedicationData(drug: Drug, encounterUuid: s
             asNeeded: false,
             asNeededCondition: '',
             startDate: new Date(),
-            duration: 0,
+            duration: null,
```
Comment (FlorianRappl) [inline src/order-basket/drug-search.ts:72]: Yeah that `null`;)

**Review by FlorianRappl — APPROVED** [review]
LGTM!

**Comment by FlorianRappl** [comment]
> Should we also increment the package version/create a new release? After all, some flows of the last release have been fixed (e.g. revising orders, having an indefinite duration, ...).

Yes 👍

File: src/order-basket/drug-search.ts (hunk 1 above)
Comment (manuelroemer) [reply src/order-basket/drug-search.ts:72]: In hindsight, that was kind of obvious 😆

**Comment by manuelroemer** [comment]
> 
> 
> > Should we also increment the package version/create a new release? After all, some flows of the last release have been fixed (e.g. revising orders, having an indefinite duration, ...).
> 
> Yes 👍

Alright, done!

**Pull request merged**